packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx

```diff
@@ -11,136 +15,158 @@ const VisualizationCardOptions: FC = () => {
     const cartesianType = (cartesianConfig.dirtyConfig?.series || [])[0]?.type;
     const cartesianFlipAxis = (cartesianConfig.dirtyConfig?.series || [])[0]
         ?.flipAxes;
+    const [isOpen, setIsOpen] = useState<boolean>(false);
+    const [activeChartType, setActiveChartType] = useState({
+        text: 'Column chart',
+        icon: 'timeline-bar-chart',
+    });
```

Comment:
Is this working for saved charts with a different type ? Looks like it will always show the column label and icon.